### ISSUE TITLE
lock file "postmaster.pid" already exists

### ISSUE THREAD

**Issue by adelosa** [tk0miya/testing.common.database#16]
testing.common.database 2.0.1 is causing the following error in my build. 
Tox is run within centos6 docker base image using testing.postgresql library.
Reran same job forcing use of testing.common.database 2.0.0 version and this error was not encountered.

```
Traceback (most recent call last):
  File "/opt/atlassian/pipelines/agent/build/test.py", line 1, in setUp
    self.postgresql = Postgresql()
  File "/opt/atlassian/pipelines/agent/build/.tox/py27/lib/python2.7/site-packages/testing/common/database.py", line 57, in __call__
    return self.target_class(**self.settings)
  File "/opt/atlassian/pipelines/agent/build/.tox/py27/lib/python2.7/site-packages/testing/common/database.py", line 98, in __init__
    self.start()
  File "/opt/atlassian/pipelines/agent/build/.tox/py27/lib/python2.7/site-packages/testing/common/database.py", line 154, in start
    self.wait_booting()
  File "/opt/atlassian/pipelines/agent/build/.tox/py27/lib/python2.7/site-packages/testing/common/database.py", line 171, in wait_booting
    self.read_bootlog())
RuntimeError: *** failed to launch Postgresql ***
FATAL:  lock file "postmaster.pid" already exists
HINT:  Is another postmaster (PID 182) running in data directory "/tmp/tmpwRIVFL/data"?

```
Let me know what additional information you need me to provide. I use your library all the time and would be happy to assist in the resolution..

**Comment by DeoLeung** [comment]
same here on debian jessie image

**Comment by vikt0rs** [comment]
@tk0miya - is it possible to release bugfix version, when the fix for this issue will be merged?

**Comment by tk0miya** [comment]
Fixed by #18. I will release this ASAP.
